extensions/amp-brightcove/0.1/test/test-amp-brightcove.js

```diff
@@ -133,14 +138,27 @@ describes.realWin(
             '/index.html?videoId=ref:amp-test-video&playsinline=true'
         );
 
+        const newSrc = new Promise((resolve) => {
+          const cb = () => {
+            resolve(iframe.src);
+            observer.disconnect();
+          };
+
+          const observer = new MutationObserver(cb);
+          observer.observe(iframe, {
+            attributes: true,
+            attributesFilter: ['src'],
+          });
+        });
+
```

Comment:
Is this necessary? I don't think your changes should have affected this.

Reply:
right, it isn't now getIframeSrc_() is synchronous again